src/main/host/shd-process.c

```diff
@@ -564,6 +572,11 @@ Process* process_new(gpointer host, guint processID,
     utility_assert(pluginName);
     proc->plugin.name = g_string_new(pluginName);
     proc->plugin.path = g_string_new(pluginPath);
+    if(pluginSymbol == NULL) {
+        proc->plugin.startSymbol = g_string_new(PLUGIN_DEFAULT_SYMBOL);
```

Comment:
Along with the previous comment, we could leave proc->plugin.startSymbol NULL here, and handle NULL strings properly in places that use it. That way we don't allocate the constant PLUGIN_DEFAULT_SYMBOL string.